### DISCUSSION TITLE
Youtube Homepage and Search page changes

### DISCUSSION THREAD

**Discussion by Exonymos** [openstyles/stylus#1757] [Q&A]
YouTube keeps changing the homepage and search page CSS, my problem is the 3 videos only in a row, they are too big for me and I want to make it 4, and all the community posts and the shorts that come between are also the same. And the search page videos are the same.
Please if someone has a way do tell, I tried writing custom CSS but it didn't work

**Homepage:**
![Ned0S9L](https://github.com/openstyles/stylus/assets/83488939/fa0d8642-cda1-454f-9bcb-85207c13aace)

**Search Page:**
![image](https://github.com/openstyles/stylus/assets/83488939/93fd3df7-5497-4153-b4aa-1e1cfc6a3e34)

**Comment by decembre** [comment]
It's a little bit Tricky:
Many different presentations of special results inside the search pages...
You want something like that? (It's only for a large screen):
![YOUTUBE WIDESCREEN - TEST Search Results Compact SMALL - 01 - 2024-04-15_161405](https://github.com/openstyles/stylus/assets/868778/66042dee-699e-4c78-afc2-96703bf590ad)

**Reply by Exonymos** [reply]
No not really but I'm just using the zoom out at 90% and it's looks fine
